src/cast_core/test/CrudActorYamlTests.yml

```diff
@@ -434,6 +434,17 @@ Tests:
       - Filter: {a: 1}
         Count: 0
 
+  - Description: simple findOne
+    Operations:
+      - OperationName: insertOne
+        OperationCommand:
+          Document: {a: 100}
+      - OperationName: findOne
+        OperationCommand:
+          Filter: {a: 100}
+    OutcomeData:
+      - {a: 2}
```

Comment:
It's asserting there is a `{a: 2}` document. You can just remove the `OutcomeData` here and it should work. There's not an easy way to assert the results of the `findOne` because the Actor doesn't expose it to the outside world.

Reply:
Yeah, I originally didn't have this and this was a shot in the dark to fix the failure. It seems to have been no difference. I'll back it back out, but it won't change the result. See results here: https://github.com/mongodb/genny/pull/364/commits/089495cc4aa1c2d69c960925365fee39a45ea077